### PR TITLE
Update VIs in TestStand Integration to store pin map id to TestStand's RunTimeVariables

### PR DESCRIPTION
**Pull request by Jayaseelan-James**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-labview/blob/main/CONTRIBUTING.md).
- [ ]  Automatically post PR comments with images for G code changes?

### What does this Pull Request accomplish?
- Updates the `Get Pin Map ID for Sequence.vi` to get and the Pin map id from the TestStand's RunTimeVariables container.
- Remove the unused VIs such as `Resolve Pin Map Path for Sequence.vi`, `Set Pin Map ID for Sequence.vi`, `Update Pin Map for Sequence.vi` and `Update Pin Map from File.vi`

### Why should this Pull Request be merged?
With the "Update pin map" step in place, the step will store the Pin map id in TestStand's RunTimeVariables container. To align with this change, the `Get Pin Map ID for Sequence.vi` is updated to get the Pin map id.

### What testing has been done?
Verified the NI-DCPower Source Voltage's example TestStand sequence with the "Update pin map" step by copying and pasting:
1. The corresponding TestStand version's INI file from this [branch](https://dev.azure.com/ni/DevCentral/_git/ASW/pullrequest/609779).
2. The [MeasurementLinkString.ini](https://dev.azure.com/ni/DevCentral/_git/ASW?path=/Source/MeasurementServices/TestStandCustomStep/Components/Language/English/MeasurementLinkStrings.ini) and TestStand202XMeasurementClient DLLs from main or you could get the latest MeasurementLink installer from argo (please make sure to have installed both MeasurementLink 2024 Q1 and MeasurementLink Support for TestStand 202X 2024 Q1). 

Note: Merging this PR will break all the examples' TestStand sequence files as they all point to this single file. All the example sequence updates will be raised in a separate PR as soon as possible.

### CODE REVIEW

**Review by pbirkhol-ni — APPROVED** [review]

**Comment by Jayaseelan-James** [comment]
@jasonmreding, @pbirkhol-ni I'm resetting your votes as I've removed _now_ unused VIs and have updated the `MeasurementLink TestStand Integration.mnu` to remove those VIs. Please consider reviewing the changes.

File: Source/Runtime/MeasurementLink TestStand Integration/MeasurementLink TestStand Integration.lvlib
Comment (pbirkhol-ni) [inline Source/Runtime/MeasurementLink TestStand Integration/MeasurementLink TestStand Integration.lvlib:1]: I think all of these VIs can be deleted too. There is also a test or two that can be deleted once you delete those VIs.
image

Comment (pbirkhol-ni) [reply Source/Runtime/MeasurementLink TestStand Integration/MeasurementLink TestStand Integration.lvlib:1]: That is fine. I would keep the 4 VIs in a virtual folder, but putting that in the Helpers directory is fine.

Comment (Jayaseelan-James) [reply Source/Runtime/MeasurementLink TestStand Integration/MeasurementLink TestStand Integration.lvlib:1]: Moved the VIs to `..\Tests.Runtime\Measurement Server\Helpers\` and added the VIs a virtual folder.

Comment (pbirkhol-ni) [reply Source/Runtime/MeasurementLink TestStand Integration/MeasurementLink TestStand Integration.lvlib:1]: Looks good!

**Review by pbirkhol-ni — APPROVED** [review]

**Review by jasonmreding — APPROVED** [review]

**Pull request merged**